ecs_fargate/tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+    unit
+    flake8
+
+[testenv]
+platform = linux|darwin|win32
+
+[testenv:unit]
+deps =
+    -e../datadog_checks_base[deps]
+    -rrequirements-dev.txt
+
+commands =
+    pip install --require-hashes -r requirements.txt
+    pytest -v
```

Comment:
More of a question, but since the file is `test_unit`, do we want to make that file as a unit test and run that explicitly with pytest -m 'unit'?

Reply:
not really needed until we introduce integration tests